$MirOS: ports/misc/mc/patches/patch-src_utilunix_c,v 1.1 2005/03/18 15:50:29 tg Exp $
$OpenBSD: patch-src_utilunix_c,v 1.1 2004/04/11 00:40:59 naddy Exp $
--- src/utilunix.c.orig	Thu Dec 26 14:47:24 2002
+++ src/utilunix.c	Fri Oct 15 17:49:34 2004
@@ -235,9 +235,9 @@ int my_system (int flags, const char *sh
 	signal (SIGCHLD, SIG_DFL);
 
 	if (flags & EXECUTE_AS_SHELL)
-	    execl (shell, shell, "-c", command, NULL);
+	    execl (shell, shell, "-c", command, (char *)NULL);
 	else
-	    execlp (shell, shell, command, NULL);
+	    execlp (shell, shell, command, (char *)NULL);
 
 	_exit (127);		/* Exec error */
     } else {
